Library/Homebrew/formula_installer.rb

```diff
@@ -489,14 +489,20 @@ def expand_dependencies(deps)
       end
     end
 
-    if pour_bottle && !Keg.relocation_formulae.include?(formula.name)
-      bottle_deps = Keg.relocation_formulae
-                       .map { |formula| Dependency.new(formula) }
-                       .reject do |dep|
+    if pour_bottle && !Keg.bottle_dependencies.empty?
+      bottle_deps = if !Keg.bottle_dependencies.include?(formula.name)
+        Keg.bottle_dependencies
+      elsif !Keg.relocation_formulae.include?(formula.name)
+        Keg.relocation_formulae
+      else
+        []
+      end
```

Comment:
@MikeMcQuaid Do you like this code better than what I had before?
To compare see the other version at https://github.com/sjackman/homebrew/commit/64467bff01c93f62bbaed8c37c67f7aedbe1a692#diff-1f60679eb76b419ad46426e2a3b50621

Reply:
Yes, much better, thanks 👍